<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-C0K6CVK" failures="4" tests="11" name="runner.MyRunner" time="59.714" errors="0" timestamp="2023-04-08T11:51:57 IST" skipped="0">
  <testcase classname="runner.MyRunner" name="runScenario" time="5.394">
    <failure message="null" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.fail(Assert.java:115)
at stepdefinitions.LoginTest.user_should_get_a_proper_warning_message(LoginTest.java:59)
at ✽.user should get a proper warning message(file:///C:/OpenCart-Selenium-Cucumber-POM/src/test/resources/featurefiles/login.feature:16)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="runner.MyRunner" name="runScenario" time="5.227"/>
  <system-out/>
  <testcase classname="runner.MyRunner" name="runScenario" time="4.837">
    <failure message="null" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.fail(Assert.java:115)
at stepdefinitions.LoginTest.user_should_get_a_proper_warning_message(LoginTest.java:59)
at ✽.user should get a proper warning message(file:///C:/OpenCart-Selenium-Cucumber-POM/src/test/resources/featurefiles/login.feature:37)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="runner.MyRunner" name="runScenario" time="4.895"/>
  <system-out/>
  <testcase classname="runner.MyRunner" name="runScenario" time="4.437"/>
  <system-out/>
  <testcase classname="runner.MyRunner" name="runScenario" time="5.036">
    <failure message="null" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.fail(Assert.java:115)
at stepdefinitions.LoginTest.user_should_get_a_proper_warning_message(LoginTest.java:59)
at ✽.user should get a proper warning message(file:///C:/OpenCart-Selenium-Cucumber-POM/src/test/resources/featurefiles/login.feature:30)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="runner.MyRunner" name="runScenario" time="4.500"/>
  <system-out/>
  <testcase classname="runner.MyRunner" name="runScenario" time="5.256">
    <failure message="null" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.fail(Assert.java:115)
at stepdefinitions.LoginTest.user_should_get_a_proper_warning_message(LoginTest.java:59)
at ✽.user should get a proper warning message(file:///C:/OpenCart-Selenium-Cucumber-POM/src/test/resources/featurefiles/login.feature:23)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="runner.MyRunner" name="runScenario" time="7.328"/>
  <system-out/>
  <testcase classname="runner.MyRunner" name="runScenario" time="6.370"/>
  <system-out/>
  <testcase classname="runner.MyRunner" name="runScenario" time="6.434"/>
  <system-out/>
</testsuite> <!-- runner.MyRunner -->
